records/2022-Q1-activity-report.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: contact
+title: Scala Center Activity Report for 2022 Q1
+---
+
```

Comment:
That one looks like this:

![Screenshot 2022-03-30 at 09-08-02 Scala Center Activity Report for 2022 Q1](https://user-images.githubusercontent.com/332812/160775798-be4849db-7d1c-43a6-a089-fb73f918505a.png)

Reply:
Additionaly, the fact that the reports are available in HTML rather than just PDF should make them more SEO-friendly.